web-console/4.0-designs/actions/actions.md

```diff
@@ -0,0 +1,43 @@
+# Actions
+
+## Design
+
+**Detail Page Actions**
+
+![Detail Page Actions 1](img/1.png)
+
+![Detail Page Actions 2](img/2.png)
+
+* Actions dropdowns should be in the top right corner of the page, in line with the page title.
+* No icon should be used on the dropdown title.
+* Action items in the dropdown should not have ellipses at the end.
+* Page tabs should not appear in the as actions in the dropdown.
+* If there is a primary action, that action can be pulled out of the actions dropdown and displayed as a button to the left of the kebab.
```

Comment:
I think this should read:  ...displayed as a button to the left of the **dropdown** (not kebab).